docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -0,0 +1,366 @@
+### Cohere Embedding Connector Blueprint:
+
+This blueprint will show you how to connect a Cohere embedding model to your Opensearch instance, including creating a k-nn index and your own Embedding pipeline. You will require a Cohere API key.
+
+Cohere currently offers the following Embedding models (model name and embedding dimensions):
+
+- embed-english-v3.0 1024
+- embed-multilingual-v3.0 1024
+- embed-english-light-v3.0 384
+- embed-multilingual-light-v3.0 384
+- embed-english-v2.0 4096
+- embed-english-light-v2.0 1024
+- embed-multilingual-v2.0 768
+
+See [Cohere's /embed API docs](https://docs.cohere.com/reference/embed) for more details.
+
+#### 1; Update your Opensearch cluster settings
+
+```json
+PUT /_cluster/settings
+
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+          "^https://production\\.api\\.cohere\\ai/.*$",
```

Comment:
Did Cohere change the endpoint ? We have ` "^https://api\\.openai\\.com/.*$",` in code now
https://github.com/opensearch-project/ml-commons/blob/main/plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java#L129C20-L129C53

If needed, please also update the code to replace or add the trusted endpoint .

Reply:
We support both `production.api.cohere.ai` and `api.cohere.ai` base URLs. I assume you mean `api.cohere.ai` from your comment, I'll change it to that just to avoid any confusion.